pkg/util/helpers.go

```diff
@@ -100,3 +100,95 @@ func getDebianVersion(osReleaseMap map[string]string) string {
 	// https://manpages.debian.org/testing/systemd/os-release.5.en.html
 	return fmt.Sprintf("%s %s", osReleaseMap["ID"], osReleaseMap["VERSION"])
 }
+
+func GetDefaultModules() ([]string, error) {
+	osReleaseMap, err := osrelease.ReadFile(osReleasePath)
+	if err != nil {
+		return nil, err
+	}
+	switch osReleaseMap["ID"] {
+	case "cos":
+		return getCOSModules(), nil
+	case "ubuntu":
+		return getUbuntuModules(), nil
+	default:
+		return nil, fmt.Errorf("Unsupported for the OS:%q", osReleaseMap["ID"])
+	}
+}
+
+func getCOSModules() []string {
```

Comment:
Once thing to consider in future is to put these into a config file instead of a helper function to make it more clean, not necessarily need to do in this PR since I am assuming that modules would have very little change (correct me if I am wrong).

Reply:
Maybe we should have a configure file right now, and I see the benefits:
1.) Different OS can use different config with the same logic
2.) Different product can use the config without changing NPD code
3.) The configuration can be changed on the fly without changing NPD code

a plain json file like known_modules.json should serve this purpose.

Aslo
4.) Different release will keep adding kernel modules, this will make things 
more complicated.

That means, a lot of modules will be classified unknown without updating NPD code.

Reply:
+ to using a separate json or yaml file here.

Reply:
added a separate json file which can be added in the config.